lib/Util/ObjectTypes.php

```diff
@@ -160,6 +160,10 @@ class ObjectTypes
      * @var array Mapping from v2 object types to resource classes
      */
     const v2Mapping = [
+        // V1 Class needed for fetching the right related object
```

Comment:
there's likely a way to fix this in codegen (like what I did for java); probably not a blocker for this but consider logging a jira ticket to revisit later.

Reply:
Already filed one